tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -233,3 +313,66 @@ def test_harvesting_arxiv_workflow_manual_accepted(
         obj = eng.processed_objects[0]
         # It was accepted
         assert obj.status == ObjectStatus.COMPLETED
+
+
+def test_workflows_change_document_type_article(workflow_app, insert_journals_in_db, workflow):
+    workflow.data = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'lit_record_article_all_journals_proceedings_and_none_refereed.json')))
+
+    set_document_type_and_refereed(workflow, None)
+
+    assert 'conference paper' in workflow.data['document_type']
```

Comment:
as this function is setting both `refereed` and `conference paper`, you should assert something about both in all tests.